main.py

```diff
@@ -10,14 +10,50 @@
 NUSCENES_ROOT = 'nuscenes'
```

Comment:
can we make `NUSCENES_ROOT` and `OUTPUT_ROOT` arguments to argparse`? Fine to keep the defaults to the values you have listed here in the `argparse` commands

Reply:
do you also mind using the PEP8 import layout? https://www.python.org/dev/peps/pep-0008/#imports
```
Imports should be grouped in the following order:

Standard library imports.
Related third party imports.
Local application/library specific imports.
You should put a blank line between each group of imports.
```

Reply:
Added in the latest commit